frame/dapps-staking/API.md

```diff
@@ -0,0 +1,228 @@
+# Pallet dapps-staking RPC API
```

Comment:
I suggest changing the file name to `README.md`

Reply:
renamed